lightning/src/ln/channel.rs

```diff
@@ -567,7 +567,9 @@ const COMMITMENT_TX_WEIGHT_PER_HTLC: u64 = 172;
 #[cfg(test)]
 pub const COMMITMENT_TX_WEIGHT_PER_HTLC: u64 = 172;
 
-/// Maximmum `funding_satoshis` value, according to the BOLT #2 specification
+pub const ANCHOR_OUTPUT_VALUE: u64 = 330;
```

Comment:
Let's add the suffix `_SATOSHIS` ? I think we do this for almost all our constified amounts and it's so easy to confuse with msat.